src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+$calendar-day-width: 100px;
+
+.calendar {
+  width: $calendar-day-width * 7 + 6px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  $days: (
+    mon: 0,
+    tue: 1,
+    wed: 2,
+    thu: 3,
+    fri: 4,
+    sat: 5,
+    sun: 6,
+  );
+
+  @each $day, $i in $days {
+    &--start-day-#{$day} {
+      .calendar__day:nth-child(1) {
+        margin-left: (
+          $calendar-day-width * ($i) + ($calendar-day-width * ($i) / 100)
```

Comment:
The calculation for `margin-left` seems to be adding an extra percentage of the `calendar-day-width`. Ensure this is intentional and aligns with your design requirements. If not, consider revising the calculation.